Gemfile

```diff
@@ -1,5 +1,10 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
+gem 'jekyll'
+gem 'jekyll-paginate'
+gem 'jekyll-redirect-from'
```

Comment:
Now that we're using Netlify, I'd like to kill this and replace them with proper server-side redirects (in `_redirects` file)